### ISSUE TITLE
Update README with how contribute instructions

### ISSUE THREAD

**Issue by nukeador** [zhukov/webogram#52]
My suggestion is that the process to contribute should be detailed on the readme.

Quick proposal:

If you want to add a new feature of fix a bug you will have to:
- Create a new issue describing the feature/bug.
- Wait for project owner to comment.
- Once the feature is approved or bug confirmed send a pull request referencing the issue number on the commit message (ex. "Grouped notifications. Fix #number)

When you do pull requests:
- Keep them requests clean. The less commits messages the better.
- It's recommended to create a new branch on your fork and keep master clean and sync with upstream repo.
